### ISSUE TITLE
Reports

### ISSUE THREAD

**Issue by brockclayton** [bluerockapps/marketplace#42]
Sales reports - order, tax, shipping, returns, coupons
Product Reports - viewed purchased
Customer Reports - customer online, customer activity, orders, rewards points, credit

**Comment by cecilisrael** [comment]
Finished by Cecil

**Comment by cecilisrael** [comment]
Skeleton is made up.  Ready to manipulate as needed.

**Comment by brockclayton** [comment]
Hey Cecil,

I made a couple of changes to the reporting stuff.

I removed $httpProvider from the config line - this is only used if we want to process http requests. I originally put it in since we will be needing to hit the image storage site over http. No need for it if we don’t have any images in the module. I also removed it from the other modules. (I will put it back into the appropriate models once I get the image storage stuff working)
I set the reports module to default to the sales order report when you first click on the reports menu.

Thanks,


brock clayton | brock@bluerockapps.com <mailto:brock@bluerockapps.com> | 403.465.2151
